### ISSUE TITLE
[Android] Moovit app is not detected 

### ISSUE THREAD

**Issue by lorenc-tomasz** [includable/react-native-map-link#249]
Hi,
on Android Moovit app is not detected properly so it's not on the list. I looked into documentation: https://moovit.com/developers/deeplinking/ and probably it will needed some native code to be added.

**Comment by tschoffelen** [comment]
Hi! No need for any native code, but it will need to be added to the library for it to show up.

If you have any interest in implementing this, here is a helpful starting point:

https://github.com/includable/react-native-map-link/blob/master/docs/add-app.md

**Comment by lorenc-tomasz** [comment]
It's on supported apps list. It stopped working. 

I have patched the library and used https://react-native-share.github.io/react-native-share/docs/share-is-package-installed to check if `com.tranzmate` package is installed for Moovit app. Simply checking if can open url is not working for Andoird and Moovit at the moment.

**Comment by lorenc-tomasz** [comment]
My patch file:

```
diff --git a/node_modules/react-native-map-link/src/constants.js b/node_modules/react-native-map-link/src/constants.js
index a74291f..1813bff 100644
--- a/node_modules/react-native-map-link/src/constants.js
+++ b/node_modules/react-native-map-link/src/constants.js
@@ -32,6 +32,10 @@ export function generatePrefixes(options) {
   };
 }
 
+export const APP_PACKAGES = {
+  moovit: 'com.tranzmate',
+}
+
 export function prefixForGoogleMaps(alwaysIncludeGoogle) {
   return isIOS && !alwaysIncludeGoogle
     ? 'comgooglemaps://'
diff --git a/node_modules/react-native-map-link/src/utils.js b/node_modules/react-native-map-link/src/utils.js
index 61dcee0..b402b18 100644
--- a/node_modules/react-native-map-link/src/utils.js
+++ b/node_modules/react-native-map-link/src/utils.js
@@ -3,8 +3,9 @@
  */
 
 import {Linking, ActionSheetIOS, Alert} from 'react-native';
+import Share from 'react-native-share';
 
-import {appKeys, isIOS} from './constants';
+import {appKeys, isIOS, APP_PACKAGES} from './constants';
 
 /**
  * Get available navigation apps.
@@ -56,11 +57,19 @@ export function isAppInstalled(app, prefixes) {
       return resolve(false);
     }
 
-    Linking.canOpenURL(prefixes[app])
-      .then((result) => {
-        resolve(!!result);
-      })
-      .catch(() => resolve(false));
+    if (!isIOS && APP_PACKAGES[app]) {
+      Share.isPackageInstalled(APP_PACKAGES[app])
+        .then((response) => {
+          resolve(!!response.isInstalled);
+        })
+        .catch(() => resolve(false))
+      } else {
+      Linking.canOpenURL(prefixes[app])
+        .then((result) => {
+          resolve(!!result);
+        })
+        .catch(() => resolve(false));
+    }
   });
 }
 
```

**Comment by tschoffelen** [comment]
Feel free to open a PR if you want this to get merged into the library.

**Comment by lorenc-tomasz** [comment]
Pushed PR for this issue: https://github.com/includable/react-native-map-link/pull/250

**Comment by lorenc-tomasz** [comment]
Hi,
@tschoffelen Can you look into the PR, please? I have fixed lint issues.

**Comment by tschoffelen** [comment]
Fixed in #250
